demo/run_bdd

```diff
@@ -126,6 +126,7 @@ AGENT_PORT=8020
 AGENT_PORT_RANGE=8020-8079
 
 echo "Preparing agent image..."
+docker build -q -t indy-base -f ../docker/Dockerfile.indy --target=indy-base .. || exit 1
```

Comment:
```suggestion
docker build -q -t indy-base -f ../docker/Dockerfile.indy --target=main .. || exit 1
```
Should this be a full ACA-Py image, rather than just the `indy-base` image?

Reply:
I gave this only a few minutes thought but my thought process was that the `indy-base` target is roughly equivalent to `von-image`. I haven't looked closely at what is layered on top of von-image in the demo and bdd images so it may make sense to just use the full ACA-Py image.